pages/create/index.js

```diff
@@ -41,28 +70,53 @@ export default function CreateFlow() {
           hours={flow.duration.hours}
           minutes={flow.duration.minutes}
           id={flow.id}
+          deleteFlow={() => deleteFlow(flow.id)}
+          setEditFormId={() => setEditFormId(flow.id)}
         />
       ))}
-      {open && (
-        <CreateFlowForm
-          setFlows={setFlows}
-          flows={flows}
-          setOpen={setOpen}
-          addFlow={addFlow}
-        />
-      )}
-      <StyledAddButton onClick={toggleOpen}>{open ? "x" : "+"}</StyledAddButton>
+      {openForm && <CreateFlowForm flows={flows} addFlow={addFlow} />}
+      {editFormId != null &&
+        flows.map((flow) =>
+          flow.id === editFormId ? (
```

Comment:
```suggestion
          flow.id === editFormId && (
```
To make it simpler, you can use the double `&&`. If you do that, you have to adjust lines 97 through 99 since it's no longer a ternary expression.